rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -3007,10 +3023,13 @@ void main_window::dropEvent(QDropEvent* event)
 		{
 			return;
 		}
+
 		for (const auto& path : drop_paths)
 		{
+			Emu.GracefulShutdown(false);
```

Comment:
lol why this one inside the loop and the other one outside?

Reply:
Less LOC changes, they are equivalent in actuality.